src/m365/commands/request.ts

```diff
@@ -174,11 +176,32 @@ class RequestCommand extends Command {
         const res = await request.execute<string>(config);
         logger.log(res);
       }
-    } 
+    }
     catch (err: any) {
       this.handleError(err);
     }
   }
+
+  private formatUrl(url: string): string {
```

Comment:
Let's rename this to `resolveUrlTokens` because we're not really formatting the URL.